index.rst

```diff
@@ -11,12 +11,35 @@
 Welcome to Flux's documentation!
 ================================
 
+Flux is a flexible framework for resource management, built for your site.
+The framework consists of a suite of projects, tools, and libraries which may be used to build site-custom resource managers for High Performance Computing centers.
+
 .. toctree::
-   :maxdepth: 3
-   :caption: Contents
+   :maxdepth: 2
+   :caption: Overview
 
-   quickstart
    contributing
+   quickstart
+
+.. toctree::
+   :maxdepth: 1
+   :caption: User Relevant RFCs
```

Comment:
Looking at this in the context of the rendered page, I think these RFCs are more relevant to developers than users:
```suggestion
   :caption: Contributor Relevant RFCs
```